docs/diagrams/CountryClass.puml

```diff
@@ -4,17 +4,23 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
+package Model {
 Class Country
 Class CountryNotesManager
 Class CountryCodeVerifier
 Class Note
 Class CountryNote
-Class AddressBook
+Class TbmManager
+Interface Model <<Interface>>
+Class ModelManager
+}
 
+ModelManager .up.|> Model
+ModelManager -down-> "1" TbmManager
+TbmManager --> "1" CountryNotesManager
+CountryNotesManager "1" -r-> "*" CountryNote
 CountryNote -up-|> Note
-CountryNote -r-> "1" Country
-CountryNotesManager "1" -up-> "*" CountryNote
-Country .up.> CountryCodeVerifier
-AddressBook -left-> "1" CountryNotesManager
+CountryNote --> "1" Country
+Country ..> CountryCodeVerifier
```

Comment:
Does Country actually have a dependency on CountryCodeVerifier? It seems like its only referencing it in an assert.

Reply:
Yea its just the assert, is it better to remove the dependency?

Reply:
Not sure actually, but on second thought to be safe it's better to leave it in bah